### PR TITLE
only logged in users can view data

### LINKED ISSUE #697: Must be logged in to view RCO
If not logged in, users only have a read-only view of data.

Clients have asked that we switch strategy - so that non-logged in users can only see the login page.  They must log in to view RCO

### PR DESCRIPTION
**Pull request by TahaKhanAbdalli**
fixes #697

### CODE REVIEW

File: src/hooks/tests/protected-route.test.tsx [hunk 1]
```diff
@@ -16,18 +16,20 @@ const resourceRoutes: ResourceRoutes = {
...
-      constants.R_ITEMS,
-      resourceRoutes
-    )
-    expect(list).toBeDefined()
-    expect(show).toBeDefined()
-    expect(create).toBeUndefined()
-    expect(edit).toBeUndefined()
-  })
+  // no longer required as unauthorized users can't see any data
+
```
Comment (IanMayo) [inline src/hooks/tests/protected-route.test.tsx:20]: Hello @TahaKhanAbdalli - shouldn't we check these objects are _not_ defined?

Comment (TahaKhanAbdalli) [reply src/hooks/tests/protected-route.test.tsx:20]: Since now there's no `user` role, therefore, we can't check these objects for any undefined role.

Comment (IanMayo) [reply src/hooks/tests/protected-route.test.tsx:20]: Aah, that makes sense :-)

**Review by IanMayo — APPROVED** [review]
Tested and working

**Pull request merged**